blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "bucket_name" {
+  description = "GCS bucket name to store the Vertex AI artifacts."
+  type        = string
+  default     = null
+}
+
+variable "dataset_name" {
+  description = "BigQuery Dataset to store the training data."
+  type        = string
+  default     = null
+}
+
+
+variable "group_iam" {
+  description = "Authoritative IAM binding for the project, in {GROUP_EMAIL => [ROLES]} format."
+  type        = map(list(string))
+  default     = {}
+  nullable    = false
+}
+
+variable "identity_pool_claims" {
+  description = "Claims to be used by Workload Identity Federation (i.e.: attribute.repository/ORGANIZATION/REPO). If a not null value is provided, then google_iam_workload_identity_pool resource will be created."
+  type        = string
+  default     = null
+}
+
+variable "labels" {
+  description = "Labels to be assigned at project level."
+  type        = map(string)
+  default     = {}
+}
+
+variable "location" {
+  description = "Location used for multi-regional resources."
+  type        = string
+  default     = "eu"
+}
+
+variable "network_config" {
+  description = "Shared VPC network configurations to use. If null networks will be created in projects with preconfigured values."
+  type = object({
+    host_project      = string
+    network_self_link = string
+    subnet_self_link  = string
+  })
+  default = null
+}
+
+variable "notebooks" {
+  description = "Vertex AI workbenchs to be deployed."
+  type = map(object({
+    owner            = string
+    region           = string
+    subnet           = string
+    internal_ip_only = optional(bool, false)
+    idle_shutdown    = optional(bool)
+  }))
+  default = {}
+}
+
+variable "prefix" {
+  description = "Prefix used for the project id."
```

Comment:
I would add a comment that the prefix can be used to specify also the env, since this is relevant on the way this blueprint will be used on a multi env ML ops.

Reply:
This is something we could consider for future releases.